### PR TITLE
Migrate fast ica encoder

### PR DESCRIPTION
**Pull request by YueLiu-jina**
import transformer encoder is base on PR https://github.com/jina-ai/jina/pull/866

### CODE REVIEW

File: encoders/numeric/FastICAEncoder/README.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# FastICAEncoder
+
+`FastICAEncoder` encodes data from an ndarray in size `B x T` into an ndarray in size `B x D`.
+
+note:
```
Comment (JoanFM) [inline encoders/numeric/FastICAEncoder/README.md:5]: that is wrong under current level, that was before, now u can pass a pickled file from which to do encoding

File: encoders/numeric/FastICAEncoder/tests/test_fasticaencoder.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import numpy as np
+import pickle
+
+
+from .. import FastICAEncoder
+from encoders.numeric.test import TestClassNumeric
```
Comment (JoanFM) [inline encoders/numeric/FastICAEncoder/tests/test_fasticaencoder.py:6]: we do not want to have these kind of dependencies, every test needs to be self contained, so every test will have to implement its tests (test_encode, test_load ....)

File: encoders/numeric/test.py [hunk 3]
```diff
@@ -0,0 +1,111 @@
+import os
```
Comment (JoanFM) [inline encoders/numeric/test.py:1]: This needs to be removed, every executor needs have its own self contained tests and no dependency from other files in the tree

**Review by JoanFM — CHANGES_REQUESTED** [review]

**Comment by JoanFM** [comment]
This is a missconception on how we have to tests executors (see comments also added in #136)

**Comment by nan-wang** [comment]
@YueLiu-jina I'm looking into this PR

**Review by hanxiao — APPROVED** [review]

**Review by hanxiao — APPROVED** [review]

File: encoders/numeric/FastICAEncoder/requirements.txt [hunk 4]
```diff
@@ -0,0 +1 @@
+sklearn
```
Comment (JamesTang-jinaai) [inline encoders/numeric/FastICAEncoder/requirements.txt:1]: should be `scikit-learn`

**Pull request merged**